pkgs/by-name/li/lianad/package.nix

```diff
@@ -8,19 +8,22 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "lianad";
-  version = "8.0"; # keep in sync with liana
+  version = "9.0"; # keep in sync with liana
```

Comment:
Wouldn't you need to update liana at the same time?

Reply:
Alright, going by the comment I would have expected something more strict. In that case the PR LGTM.

Reply:
Thanks, yeah apologies it's not super clear. I'll look at making the comment clearer in a future version bump.